src/components/Tasks/ColoredRadio.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Same comment as before, we use a slightly different convention: `import * as React from 'react';`